examples/ibmmq.yml

```diff
@@ -0,0 +1,23 @@
+integrations:
+  - name: nri-flex
+    interval: 30s
+    config:
+      name: ibmmq
```

Comment:
is this telemetry something that could be also scraped using the [ibmmq integration ](https://docs.newrelic.com/install/ibm-mq/)? If so , there is any difference on the data collected ?